src/App.js

```diff
@@ -55,7 +55,7 @@ class App extends React.Component {
   };
 
   toggleItem = id => {
-    console.log(id);
+    // console.log("toggle");
     this.setState({
       lists: this.state.lists.map(item => {
         if (item.id === id) {
```

Comment:
we dont need to do this in state, why dont we just find the ID and then change the completed property of that object?

then in our todoList component run an if statement to give it a className of completed if your completed property is true. youve already dont this so all we need to do is that first part.